rust-symcrypt/src/block_ciphers.rs

```diff
@@ -0,0 +1,17 @@
+use symcrypt_sys;
+
+
+// WIP still working on this
+pub enum BlockCipherType {
+    AES(symcrypt_sys::PCSYMCRYPT_BLOCKCIPHER),

```

Comment:
This still ends up leaking the `PCSYMCRYPT_BLOCKCIPHER` type in the public API. I think what I'd do instead is just have just `AES`, and then have some private function that converts a `BlockCipherType` into a `PSYMCRYPT_BLOCKCIPHER`.

Reply:
Changed